From 6dc415bb238d7c515f40305248d03234be88cadf Mon Sep 17 00:00:00 2001
From: popcornmix <popcornmix@gmail.com>
Date: Mon, 6 Apr 2020 17:07:31 +0100
Subject: [PATCH] vc4_hdmi_phy: Fix typo in phy_get_cp_current

This is stored in a 6-bit register field which causes a WARN

Signed-off-by: popcornmix <popcornmix@gmail.com>
---
 drivers/gpu/drm/vc4/vc4_hdmi_phy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/vc4/vc4_hdmi_phy.c
+++ b/drivers/gpu/drm/vc4/vc4_hdmi_phy.c
@@ -182,7 +182,7 @@ static u8 phy_get_cp_current(unsigned lo
 	if (vco_freq < 3700000000ULL)
 		return 0x1c;
 
-	return 0xc8;
+	return 0x18;
 }
 
 static u32 phy_get_rm_offset(unsigned long long vco_freq)
